Src/Base/AMReX_CudaAsyncFab.cpp

```diff
@@ -57,10 +59,13 @@ AsyncFab::clear ()
     if (inLaunchRegion())
     {
         if (m_impl != nullptr) {
-// CUDA 10        AMREX_GPU_SAFE_CALL(cudaLaunchHostFunc(Device::cudaStream(), amrex_devicefab_delete, p));
-            AMREX_GPU_SAFE_CALL(cudaStreamAddCallback(Device::cudaStream(),
-                                                      amrex_devicefab_delete,
-                                                      m_impl, 0));
+// CUDA 10
+            cudaHostFn_t clear = amrex_devicefab_delete;
+            AMREX_GPU_SAFE_CALL(cudaLaunchHostFunc(Device::cudaStream(), clear, m_impl));
+// CUDA 9
+//            AMREX_GPU_SAFE_CALL(cudaStreamAddCallback(Device::cudaStream(),
+//                                                      amrex_devicefab_delete,
+//                                                      m_impl, 0));
```

Comment:
Need to make sure it works for CUDA 9.

Reply:
Looks like it's fixed now. Tried the LinearSolver tutorial on with CUDA 9 using pgi and gnu. Both run as expected.